quickstart/aspnet-core/infra/main.bicep

```diff
@@ -0,0 +1,70 @@
+targetScope = 'subscription'
+
+// The main bicep module to provision Azure resources.
+// For a more complete walkthrough to understand how this file works with azd,
+// see https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/make-azd-compatible?pivots=azd-create
```

Comment:
Remove `en-us/`

Reply:
removed the comments in main.bicep to keep the content minimal.